src/app/submit.service.ts

```diff
@@ -1,20 +1,34 @@
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
-import { Observable, of } from 'rxjs';
```

Comment:
All the changes you are making in this PR that's related to the service, you can move those changes to PR#9 ?